example/cmd/PasswordGenerator.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/bitfield/qrand"
+)
+
+func main() {
+
+	var random = rand.New(qrand.NewSource())
+
+	// nchars stores the password length
+	var nchars = 15
+	charSet := []byte{'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'}
```

Comment:
We'd like to avoid having to store the alphabet, wouldn't we? Suppose we started with `a`, and we _added_ our random number to it to generate the rune we want.